src/Features/CSharp/CSharpFeaturesResources.resx

```diff
@@ -228,6 +228,9 @@
   <data name="Deprecated" xml:space="preserve">
     <value>deprecated</value>
```

Comment:
1. I have seen an issue about being consistent in ending exception messages with a dot. What is the user value of that?

2. They are not unnecessarily different, the fact that we have two similar attributes is unnecessary.

3. "Never the design" - the design is not correct. .NET was not designed to be open-source, but then it became. Adding all attributes is a very good idea, it shows that custom attributes is a not a mistake but a first-class citizen. Not showing any attributes but [Obsolete] *by default* making it configurable is a much better idea, granted.